_episodes/07-errors.md

```diff
@@ -186,7 +186,7 @@ it *always* means that there is a problem with how your code is indented.
 > because they are both [whitespace]({{ page.root }}/reference/#whitespace).
 > The Jupyter notebook actually gives us a bit of a hint,
 > but not all Python editors will do that.
-> In the following example,
+> Copy and paste the following example, where
```

Comment:
I'd prefer to leave the beginning of the sentence as it was before (`In the following example,`) because without specific direction as to *where* to paste it in, instructions seem unclear. My suggestion would be to add something like `(be sure to copy and paste the example into your interpreter instead of typing it in to preserve Tabs)`

Reply:
Makes sense, thanks!